blocks/elastic-content-strip/elastic-content-strip.js

```diff
@@ -0,0 +1,28 @@
+import { a } from '../../scripts/dom-helpers.js';
+
+import {
+  decorateInnerHrefButtonsWithArrowIcon,
+  decorateBlockMode,
+} from '../../scripts/scripts.js';
+
+export default function decorate(block) {
+  block.querySelectorAll('.elastic-content-strip > div > div').forEach((div) => {
+    const linkHref = div.querySelectorAll('a')[0].href;
```

Comment:
If you just want the first you can just use:
```suggestion
    const linkHref = div.querySelector('a').href;
```